interface/super/edit_list.php

```diff
@@ -1,1105 +1,1082 @@
+    return $s;
   }
-  else if ($list_id == 'code_types') {
-    $res = sqlStatement("SELECT * FROM code_types " .
-      "ORDER BY ct_seq, ct_key");
-    while ($row = sqlFetchArray($res)) {
-      writeCTLine($row);
-    }
-    for ($i = 0; $i < 3; ++$i) {
-      writeCTLine(array());
+  
+  // Write a form line as above but for the special case of Code Types.
+  //
+  function writeCTLine($ct_array) {
+    global $opt_line_no,$cd_external_options;
+  
+    ++$opt_line_no;
+    $bgcolor = "#" . (($opt_line_no & 1) ? "ddddff" : "ffdddd");
+  
+    echo " <tr bgcolor='$bgcolor'>\n";
+  
+    echo ctGenCBox($opt_line_no, $ct_array, 'ct_active',
+      xl('Is this code type active?'));
+    echo ctGenCell($opt_line_no, $ct_array, 'ct_key' , 6, 15,
+      xl('Unique human-readable identifier for this type'));
+    echo ctGenCell($opt_line_no, $ct_array, 'ct_id'  , 2, 11,
+      xl('Unique numeric identifier for this type'));
+    echo ctGenCell($opt_line_no, $ct_array, 'ct_label' , 6, 30,
+      xl('Label for this type'));
+    // if not english and translating lists then show the translation
+    if ($GLOBALS['translate_lists'] && $_SESSION['language_choice'] > 1) {
+         echo "  <td align='center' class='translation'>" . xlt($ct_array['ct_label']) . "</td>\n";
     }
+    echo ctGenCell($opt_line_no, $ct_array, 'ct_seq' , 2,  3,
+      xl('Numeric display order'));
+    echo ctGenCell($opt_line_no, $ct_array, 'ct_mod' , 1,  2,
+      xl('Length of modifier, 0 if none'));
+    echo ctGenCell($opt_line_no, $ct_array, 'ct_just', 4, 15,
+      xl('If billing justification is used enter the name of the diagnosis code type.'));
+    echo ctGenCell($opt_line_no, $ct_array, 'ct_mask', 6,  9,
+      xl('Specifies formatting for codes. # = digit, @ = alpha, * = any character. Empty if not used.'));
+    echo ctGenCBox($opt_line_no, $ct_array, 'ct_claim',
+      xl('Is this code type used in claims?'));
+    echo ctGenCBox($opt_line_no, $ct_array, 'ct_fee',
+      xl('Are fees charged for this type?'));
+    echo ctGenCBox($opt_line_no, $ct_array, 'ct_rel',
+      xl('Does this type allow related codes?'));
+    echo ctGenCBox($opt_line_no, $ct_array, 'ct_nofs',
+      xl('Is this type hidden in the fee sheet?'));
+    echo ctGenCBox($opt_line_no, $ct_array, 'ct_proc',
+      xl('Is this a procedure/service type?'));
+    echo ctGenCBox($opt_line_no, $ct_array, 'ct_diag',
+      xl('Is this a diagnosis type?'));
+    echo ctGenCBox($opt_line_no, $ct_array, 'ct_term',
+      xl('Is this a Clinical Term code type?'));
+    echo ctGenCBox($opt_line_no, $ct_array, 'ct_problem',
+      xl('Is this a Medical Problem code type?'));
+    echo ctGenCBox($opt_line_no, $ct_array, 'ct_drug',
+      xl('Is this a Medication type?'));
+    echo ctSelector($opt_line_no, $ct_array, 'ct_external',
+      $cd_external_options, xl('Is this using external sql tables? If it is, then choose the format.'));
+    echo " </tr>\n";
   }
-  else if ($list_id == 'issue
```

Comment:
Why did you choose 3-1-1 to load it was using 1-2-1.

Reply:
Some of the programs can be upgraded on the jquery but they require testing to make sure that the change is not causing an issue.

Reply:
Art asked me to upgrade jquery
Let the tests be made and if something happens i downgrade it